plugins/metadata/dc11/filter/Dc11SchemaPublicationFormatAdapter.inc.php

```diff
@@ -180,8 +180,28 @@ function extractMetadataFromDataObject(&$publicationFormat) {
 		$this->_addLocalizedElements($dc11Description, 'dc:source', $sources);
 
 		// Language
+		$submissionLanguage = $publication->getData('locale');
```

Comment:
This will need adaptation for the `master` branch when it's ported there, I think, as the `locale` column has moved from `publications` to `submissions`.